Julia/math/automorphic_number.jl

```diff
@@ -0,0 +1,51 @@
+"""Julia program to check if a number is automorphic or not.
+An Automorphic Number is such a number whose square ends in the same digits as  the number itself."""
+
+
+function check_automophic(num)
+    temp = num * num
+
+    while(num > 0)
+        # Extract the last digits of the given number and its square.
+        lastNum = num % 10
+        lastSquare = temp % 10
+
+        # Check if they are equal
+        if(lastNum != lastSquare)
+            return false
+        end
+
+        num = num ÷ 10
+        temp = temp ÷ 10
+    end
+
+    return true
+end
+
+print("Enter the number: ")
+n = readline()
+n = parse(Int, n)
+res = check_automophic(abs(n))
```

Comment:
It would be better if the driver code returns the final value than you writing conditional statements for classifying the output. The main function should always be the driver function.

Reply:
Currently the driver function return the final value and the conditional statements are used for showing the right output statement. 

Should I change the code such that the `check_automophic` function returns the string to be printed?

Reply:
Yes, that would look better. The change I have asked is not a major one but it is a good coding practice hence thought of bringing it up for you :)

Reply:
Done